docs/advanced/payments/adyen.mdx

```diff
@@ -470,3 +516,17 @@ the Cart amount and the Store currency. Different contexts could display
 different payment methods in the checkout.
 
 :::
+
+# Payment methods support
```

Comment:
```suggestion
## Payment methods support
```

Reply:
I found it a bit weird to have this on the same level as the other "As a XXX", but I can go with this.